### PR TITLE
Adding max-gas-price var for txn

### LINKED ISSUE #19: Add max gas fee limit `variable`.
@mikeghen any thoughts? I seem to have burnt 44 MATIC due to sudden gas fee spikes, waiting for 30 mins for better deal won't be an issue.

### PR DESCRIPTION
**Pull request by shreyaspapi**
fixes #19

### CODE REVIEW

File: dags/blocksec_plugin/contract_interaction_operator.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 from blocksec_plugin.web3_hook import Web3Hook
 from blocksec_plugin.ethereum_wallet_hook import EthereumWalletHook
 from web3.exceptions import InvalidAddress
+from constants.constants import PriceConstants
```
Comment (mikeghen) [inline dags/blocksec_plugin/contract_interaction_operator.py:6]: This is a great idea, maybe we can refactor all the constants, `AddressConstants` for example

File: dags/blocksec_plugin/contract_interaction_operator.py [hunk 2]
```diff
@@ -62,7 +65,7 @@ def execute(self, context):
         raw_txn = self.function(**self.function_args)\
                              .buildTransaction(dict(
                                nonce=int(self.nonce),
-                               gasPrice = int(max(33,self.web3.eth.gasPrice) *\
+                               gasPrice = int(max(self.max_gas_price, self.web3.eth.gasPrice) *\
```
Comment (mikeghen) [inline dags/blocksec_plugin/contract_interaction_operator.py]: Should this become `min` won't the gas price blowing up always select the larger of the two, which would be `gasPrice` not `max_gas_price` which would be the smaller of the two

File: dags/blocksec_plugin/contract_interaction_operator.py (hunk 1 above)
Comment (mikeghen) [reply dags/blocksec_plugin/contract_interaction_operator.py:6]: but don't do that in this PR, this is just a comment for later

**Review by mikeghen — APPROVED** [review]

File: dags/blocksec_plugin/contract_interaction_operator.py (hunk 2 above)
Comment (shreyaspapi) [reply dags/blocksec_plugin/contract_interaction_operator.py]: Yes that correct good catch!

**Pull request merged**